config_flow.py

```
@@ -42,32 +59,108 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
         
         if user_input is not None:
+            data = {
+                CONF_HOST: self._host,
+                CONF_AUTH_METHOD: AUTH_METHOD_ACCESS_CODE,
+                CONF_ACCESS_CODE: user_input[CONF_ACCESS_CODE],
+            }
+            
             try:
-                info = await validate_input(self.hass, user_input)
+                info = await validate_input(self.hass, data)
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)